posthog/api/test/test_session.py

```diff
@@ -34,30 +34,40 @@ def test_expected_session_properties(self):
         expected_properties = {
             "$autocapture_count",
             "$channel_type",
-            "$end_timestamp",
-            "$entry_current_url",
-            "$entry_pathname",
-            "$entry_hostname",
             "$end_current_url",
-            "$end_pathname",
             "$end_hostname",
+            "$end_pathname",
+            "$end_timestamp",
+            "$entry__kx",
```

Comment:
I had sorted these in alphabetical order in my PR, guess we'll just merge them like this :)